boskos/cmd/cli/cli.go

```diff
@@ -93,6 +101,9 @@ scripts with a simple interface.`,
 		Args: cobra.NoArgs,
 	}
 	root.PersistentFlags().StringVar(&options.serverURL, "server-url", "", "URL of the Boskos server")
+	root.PersistentFlags().StringVar(&options.username, "username", "", "Username used to access the Boskos server")
+	root.PersistentFlags().StringVar(&options.passwordFile, "password-file", "", "The path to password file used to access the Boskos server")
+	root.PersistentFlags().BoolVar(&options.insecureSkipVerify, "insecure-skip-tls-verify", false, "If true, the server's certificate will not be checked for validity. This will make your HTTPS connections insecure")
```

Comment:
Why?

Reply:
https://github.com/kubernetes/test-infra/pull/15458#discussion_r353519838

Reply:
Look at the  test in `interrupts_test.go` for an example of how to not have this problem.